### ISSUE TITLE
play state chack doesn't use theme color

### ISSUE THREAD

**Issue by Ewillian9** [loof2736/scyfin#25]
is it intended?
![image](https://github.com/loof2736/scyfin/assets/156851131/0b27eb5b-4328-4f50-969f-4c96216f0d6c)

**Comment by loof2736** [comment]
Yes, it is intended for better readability so you can easily differenciate between the two. If you want it to respect the theme, just add this below the rest of the CSS:

.playedIndicator { background: var(--primary-accent-color) !important; }

**Comment by Ewillian9** [comment]
I see, thanks for the explanation and the css
